about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/f25b74c76b.js" crossorigin="anonymous"></script>
+    <title>capstone-project</title>
+</head>
+<body>
+  <header id="dsktop-header">
+    <nav id="nav1" title="first-navigation">
+      <ul>
+        <li><a class="fa-brands fa-facebook-f"></a></li>
+        <li><a class="fa-brands fa-twitter"></a></li>
+        <li><a href="#" class="nav1item">English</a></li>
+        <li><a href="#" class="nav1item">My Page</a></li>
+        <li><a href="#" class="nav1item">Logout</a></li>
+      </ul>
+    </nav>
+    
+    <nav id="nav2" title="second-navigation"> 
+      <a class="main-logo" href="#about"><img src="images/icecream.png" alt=""></a> 
+      <a class="nav2Item" href="./index.html">Home</a>
+      <a class="nav2Item" href="./index.html">Packages</a>
+      <a class="nav2Item" href="#join">Join</a>
+      <a class="nav2Item" href="#sponsor">Sponsor</a>
+      <a class="nav2Item" href="#news">Recipes</a>
+      <a class="mainItem" href="./index.html">workshops</a> 
+    </nav>   
+</header>
+
+<header id="mobile-header">
+  <nav class="mobile-nav"> 
+      <ul onclick="menuPopup()" id="navList" title="navigation-list">
```

Comment:
- [ ] You did great implementing the mobile menu. :+1: :100: But following the best practices, all JavaScript code should be in the `.js` file.
- - I'll suggest you remove the onClick event from the HTML, then select the element using `getElemenet  or querySelector` in your main.js file, then add an event listener to it. :+1: :smile: